scalahost/nsc/src/test/scala/scala/meta/tests/scalahost/mirrors/SemanticSuite.scala

```diff
@@ -534,9 +535,11 @@ class SemanticSuite extends DatabaseSuite(SemanticdbMode.Slim) {
       new Object().<<toString>>
       List(1).<<toString>>
     }
-    """, { implicit db => (objectToString, listToString) =>
-      assert(objectToString.symbol.denot.isJavaDefined)
-      assert(!listToString.symbol.denot.isJavaDefined)
+    """, { (db, objectToString, listToString) =>
+      val denot1 = db.symbols.find(_.sym == objectToString).get.denot
```

Comment:
Let's discuss a good way to expose this API. I'd probably like to do without implicits this time.